coronavirus_biogrid.py

```diff
@@ -115,42 +116,57 @@ def import_data(data, source, version, gene_level=False, form='tab2'):
                     if gene_level:
                         g.write(evaLink(gene1, "GeneNode",gene2,"GeneNode", "interacts_with", symmetric=True, stv=stv))
 
+                    if taxonomy_id_1 == 2697049:
+                        covid_genes.append(gene1)
+                        f.write(
+                            evaLink(gene1, "GeneNode", taxonomy_id_1, "ConceptNode", "from_organism", prefix2="ncbi:"))
+                        f.write(evaLink(prot1, "MoleculeNode", taxonomy_id_1, "ConceptNode", "from_organism",
+                                        prefix1="Uniprot:", prefix2="ncbi:"))
+                        if gene_level:
+                            g.write(evaLink(gene1, "GeneNode", taxonomy_id_1, "ConceptNode", "from_organism",
+                                            prefix2="ncbi:"))
+                            g.write(evaLink(prot1, "MoleculeNode", taxonomy_id_1, "ConceptNode", "from_organism",
+                                            prefix1="Uniprot:", prefix2="ncbi:"))
+                    if taxonomy_id_2 == 2697049:
+                        covid_genes.append(gene2)
```

Comment:
this code looks similiar to the code above, so i suggest to make a function from it

Reply:
Please check the latest commit